### PR TITLE
[Fix] Both forms app send 1 push log for every character typed in userId field

### PR DESCRIPTION
**Pull request by russelarms**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the UWP implementation?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Use either Contoso.Forms.Puppet or Contoso.Forms.Demo on Android or iOS.
Type characters in userId.

Expected: push log sent only when done typing.
Actual: 1 push log sent for every typed character.

## Related PRs or issues

[AB#64145](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/64145)

### CODE REVIEW

**Comment by russelarms** [comment]
I removed bindings as EntryCell has no Unfocus event to subscribe.

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace Contoso.Forms.Demo
```
Comment (MatkovIvan) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs]:
```suggestion

namespace Contoso.Forms.Demo
```

File: Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Contoso.Forms.Demo
+{
+    [Android.Runtime.Preserve(AllMembers = true)]
```
Comment (guperrot) [inline Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs]: Was the symbol removed in release?, it's suprising as there is no reflection here so should not be necessary.

Did you actually test in release and had a crash when this attribute was missing on this constant?

Comment (MatkovIvan) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs]: @guperrot I guess it's just to make the things the same as here
https://github.com/microsoft/appcenter-sdk-dotnet/pull/1027/files#diff-3be604fc8d5e709029be52313f8b20b3R6

Comment (guperrot) [reply Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/Constants.cs]: Let's not introduce the annotation if not necessary, please run locally in release and verify.

**Pull request merged**